conf/resources/data/GB000060/18IT02110010006A10/response.json

```diff
@@ -0,0 +1,5 @@
+{
+  "MRN": "18IT02110010006A10",
+  "ResponseCode":7,
+  "ErrorDescription":"Guarantee not valid"
```

Comment:
This should be 'Office of Transit does not belong to country' as per the expected responses